README.md

```diff
@@ -39,14 +39,18 @@ provided the file system(s) also support these features.
 
 ## INSTALLATION
 
-Latest version of rdiff-backup is available from various sources. Follow the
-instructions for your specific platform.
+In older Linux distributions the rdiff-backup versions are of the 1.x series, which is not recommended for new installs anymore. Follow the instructions below to install the latest 2.x release of rdiff-backup.
 
-### Ubuntu (From PPA)
+### Ubuntu Focal or Debian Bullseye or newer (has 2.0)
 
 ```
-sudo apt update && sudo apt install software-properties-common
-sudo add-apt-repository ppa:rdiff-backup/rdiff-backup-development
+sudo apt install rdiff-backup
+```
+
+### Ubuntu backports for older versions (needs a backported 2.0)
+
+```
+sudo add-apt-repository ppa:rdiff-backup/rdiff-backup-backports
```

Comment:
@ikus060 Did you read this line when you reviewed? Now it is clear: this stable PPA is recommended first and the development PPA only at the end for development/debug purposes.